docs/comparisons.md

```diff
@@ -26,6 +26,7 @@ docs! We don't want to be biased, but it's easy that we tend to be:
 | Requires programming                     | No                               | No                              | Yes, JS       |
 | Requires templates to have a suffix      | Yes by default, configurable[^3] | No, not configurable            | You choose    |
 | Task hooks                               | Yes                              | Yes                             | Yes           |
+| Context hooks                            | Yes[^5]                          | Yes                             | ?             |
```

Comment:
I searched through the docs but honestly have no idea if it's possible to alter the user's answers (and/or add variables) before rendering with Yeoman generators.

Reply:
Well, if somebody with more experience in Yeoman comes in, let them fix this.